### ISSUE TITLE
(2.14.3) (Minor) Labs - Nightshift - Fails to stay on.

### ISSUE THREAD

**Issue by shadowbottle** [TryGhost/Ghost#10502]
Ghost 2.14.3
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic
Node v10.15.1 (recommended)
Browsers: 
Chrome: Version 72.0.3626.96 (Official Build) (64-bit)
Firefox: 67.0a1 (2019-02-14) (64-bit)

I have two instances of Ghost 2.14.3 running, prior to updating Nightshift (beta feature) would enable in the Labs section and stay enabled. If I refresh, log out/in, or open in new browser tab using same cookie, the setting is not maintained. Can still create new posts, can update the user details (does not appear to be related to general db issue).

STR:
1) log in
2) go to labs section
3) place check/enable Nightshift
3a) Observe, Nightshift is enabled
4) go about business, Nightshift will stay enabled
5) Refresh page; Nightshift fails, reverts back to Dayshift (??) (light background)
5or) Log out of admin panel, log into admin panel; Nightshift no longer active, reverts back to light background.
5or) Open new browser tab with URL for admin panel, using active cookie, Nightshift no longer active, reverted back to light background.

Basically just a heads up, I know it's a beta feature and low priority. I do find myself incredibly partial to it, so I hope it gets fixed >_> hehe <3 <3 <3

**Comment by vikaspotluri123** [comment]
This was fixed in #10499 and will be part of the next release 😉

**Comment by shadowbottle** [comment]
Oh sweet! Ok, great to hear, thanks for the information! :D

**Comment by kevinansfield** [comment]
Closing as the fix is already in master. Release is likely to be next Tuesday 🙂
